python_training/trouper.py

```diff
@@ -0,0 +1,46 @@
+
+class Number(int):
+    """
+    An object that extends the int object and adds functionality over the dividers of a number.
+    """
+    def __init__(self, number):
+        """
+        Instantiates a number object and prepares the dividers list.
+        :param number:
+        """
+        super().__init__()
+        self.number = number
+        self.dividers = [1, number]
+        self._get_all_dividers()
+
+    def _get_all_dividers(self):
+        """
+        Appends to the dividers list the rest of the numbers dividers(that are not 1 or the number itself).
+        """
+        square_root = self.number**0.5
+        for i in range(2, int(square_root) + 1):
```

Comment:
Why do you start from 2?
Also `math.ceil` does almost the same as `int(square_root) + 1`